internal/tray-agent/agent/client/agent-client.go

```diff
@@ -104,16 +106,19 @@ func (ctrl *AgentController) StopCaches() {
 	}
 }
 
-/*acquireKubeconfig sets the LIQO_KCONFIG env variable.
+/*acquireKubeconfig sets the EnvLiqoKConfig env variable.
```

Comment:
Environment variable should be uppercase and snake case

Reply:
yes, you're right. In this case *EnvLiqoKConfig* is the name of the Golang const that contains the actual environment variable name. I made this change in the comment so that when developers display this documentation in a IDE or in the [golang doc site](https://pkg.go.dev/), this information automatically contains a link to the constant itself.

Reply:
if you think this could be misleading, I could rephrase as *"sets the env variable named after EnvLiqoKConfig"* or something like that